internal/commands/relationship.go

```diff
@@ -236,15 +237,25 @@ func readRelationships(cmd *cobra.Command, args []string) error {
 
 			console.Println(string(prettyProto))
 		} else {
-			if msg.Relationship.Subject.OptionalRelation != "" {
-				console.Printf("%s:%s %s %s:%s#%s\n", msg.Relationship.Resource.ObjectType, msg.Relationship.Resource.ObjectId, msg.Relationship.Relation, msg.Relationship.Subject.Object.ObjectType, msg.Relationship.Subject.Object.ObjectId, msg.Relationship.Subject.OptionalRelation)
-			} else {
-				console.Printf("%s:%s %s %s:%s\n", msg.Relationship.Resource.ObjectType, msg.Relationship.Resource.ObjectId, msg.Relationship.Relation, msg.Relationship.Subject.Object.ObjectType, msg.Relationship.Subject.Object.ObjectId)
+			relString, err := relationshipToString(msg.Relationship)
+			if err != nil {
+				return err
 			}
+			console.Println(relString)
 		}
 	}
 }
 
+func relationshipToString(rel *v1.Relationship) (string, error) {
```

Comment:
Add a unit test for this, since its non-obvious

Reply:
addressed